docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+
+---
+title: Configuration
+---
+
+vSphere Docker Volume Service consists of two components the VIB and Docker plugin. The Docker plugin requires configuration which is described in this section
+
+The Docker volume plugin at the moment supports vSphere and Photon platforms. All the volume provisioning and management operations are supported on both the platforms. Both the platforms can be used to provision VMDK based volumes and can be attached to containers. 
+
+The configurations used by a driver while performing operations are read from a JSON file and the default location where it looks for it /etc/docker-volume-vsphere.conf. You can also override it to use  a different configuration file by providing --config option and the full path to the file. Finally the parameters passed on the CLI override the one from the configuration file.
+
+## Docker Volume Driver
+
+The vSphere volume driver can be used on a standalone or cluster of ESXi servers via the ESXi service. ESXi service needs to be installed and running on all servers. The volumes are created on ESXi host using the VIM (Virtual Infrastructure Management) APIs.
+
+The configuration for  Docker volume driver require the driver type which can be one of vsphere or vmdk (For backwards compatibility). You can also provide the log related configurations.
+
```

Comment:
Its confusing to the reader why we have "vSphere volume driver" and "docker volume driver". Please ensure carefully that a single name is used all over the document for each entity. vSphere Volume Driver is the name for the managed plugin that runs inside the VM to provision volumes.